<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-11-27T22:08:03 IST" name="tester" finished-at="2022-11-27T22:11:02 IST" duration-ms="179761">
    <groups>
    </groups>
    <test started-at="2022-11-27T22:08:03 IST" name="banking app Test" finished-at="2022-11-27T22:11:02 IST" duration-ms="179761">
      <class name="inetbanking_testcases.LoginDDT_002">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:inetbanking_testcases.LoginDDT_002@1ce24091]" started-at="2022-11-27T22:08:03 IST" name="setup" finished-at="2022-11-27T22:08:58 IST" duration-ms="55249" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginDDT(java.lang.String,java.lang.String)[pri:0, instance:inetbanking_testcases.LoginDDT_002@1ce24091]" started-at="2022-11-27T22:08:59 IST" name="loginDDT" data-provider="LoginData" finished-at="2022-11-27T22:10:36 IST" duration-ms="96768" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr456628]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[AzedupU]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method signature="loginDDT(java.lang.String,java.lang.String)[pri:0, instance:inetbanking_testcases.LoginDDT_002@1ce24091]" started-at="2022-11-27T22:10:36 IST" name="loginDDT" data-provider="LoginData" finished-at="2022-11-27T22:11:02 IST" duration-ms="26440" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr83460]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qAbUzyj]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at inetbanking_testcases.LoginDDT_002.loginDDT(LoginDDT_002.java:60)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:inetbanking_testcases.LoginDDT_002@1ce24091]" started-at="2022-11-27T22:11:02 IST" name="tearDown" finished-at="2022-11-27T22:11:02 IST" duration-ms="178" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- inetbanking_testcases.LoginDDT_002 -->
    </test> <!-- banking app Test -->
  </suite> <!-- tester -->
</testng-results>
